docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - deploying-grafana-mimir/
+description: Learn how to deploy Grafana Mimir with puppet.
+keywords:
+  - Mimir deployment
+  - Puppet
+menuTitle: Deploy with Puppet
+title: Deploy Grafana Mimir with Puppet
+weight: 13
+---
+
+# Deploy Grafana Mimir with Puppet
+
+A puppet module is community maintained to deploy Grafana mimir.
```

Comment:
Does using Puppet imply that Mimir will be installed from deb/rpm? If so, is that worth mentioning?

Reply:
Indeed, it worth mentioning.